ecosystem/typescript/aptos-cli/bin/aptos

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+const { spawn } = require("child_process");
+const fs = require("fs");
+const os = require("os");
+
+const PNAME = "aptos-cli";
+const VERSION = "2.0.2";
```

Comment:
feels like we could just host something on aptos-dev that says the latest version.

Reply:
If we wanted latest you can just use the GH API to look it up, though I think this is by design. This way we can verify each new CLI version works. You'd think it's not necessary but 2.0.3 doesn't work due to this OpenSSL stuff, so it's probably good to be careful.

Reply:
@banool does https://github.com/aptos-labs/aptos-core/pull/9961 should help with it?

Reply:
Yeah this particular issue is fixed now. My point is though that using the latest CLI version by default might be slightly risky business. We could take it for a spin though.